studio/data/auth/auth-config-query.ts

```diff
@@ -0,0 +1,61 @@
+import { useQuery, useQueryClient, UseQueryOptions } from '@tanstack/react-query'
+import { components } from 'data/api'
+import { get, isResponseOk } from 'lib/common/fetch'
```

Comment:
could we use `get` from data/fetchers instead? can refer to `branch-create-mutation.ts` for reference
we're looking to shift our get, post, patch etc calls to data/fetchers as they'll automatically type the body and response for us :)

Reply:
Fixed.